src/sidebar/components/ShareDialog/test/ExportAnnotations-test.js

```diff
@@ -42,7 +42,9 @@ describe('ExportAnnotations', () => {
     fakeDownloadJSONFile = sinon.stub();
     fakeStore = {
       defaultAuthority: sinon.stub().returns('example.com'),
-      isFeatureEnabled: sinon.stub().returns(true),
+      isFeatureEnabled: sinon
+        .stub()
+        .callsFake(ff => ff === 'client_display_names'),
```

Comment:
This makes sure existing tests keep passing, otherwise there's now two `SelectNext`s and every test asserting on that needs to be edited to filter the right one.

Reply:
This code turns off every feature other than `client_display_names` whereas the intent is to turn off `export_formats`. I think it would be better to add a `data-testid` to the two `SelectNext`s and change existing references to use that instead of the component name though. I don't think that should be too difficult?

Reply:
Unfortunately `SelectNext` does not accept data attribute. Probably something that should change, but in the interim, there might be some other way to target one specific SelectNext.

Reply:
> Unfortunately SelectNext does not accept data attribute. 

You mean our types don't support a `data-testid` for it? That seems like something we should address globally. One quick shortcut would be to add `data-testid` to the set of global props that are allowed on any JSX element (like `ref`, `key` etc.)

Reply:
Actually, it turns out that `data-testid` already is allowed on any JSX element. This is because kebab-cased property names are exempted from the usual check for property names being valid, per https://stackoverflow.com/a/67716249/434243.

Reply:
In this JSX, only the `fooBar` assignment throws an error:

```jsx
<SelectNext
  value={selectedUser}
  onChange={setSelectedUser}
  buttonId={userSelectId}
  buttonContent={
    <UserAnnotationsListItem userAnnotations={selectedUser} />
  }
  data-testid="select-user"
  foo-bar="baz"
  fooBar="baz"
/>
```

Reply:
Aha! 😃